shared/Microsoft.Extensions.WebEncoders.Sources/WebEncoders.cs

```diff
@@ -96,144 +172,185 @@ public static byte[] Base64UrlDecode(string input, int offset, int count)
         /// </remarks>
         public static byte[] Base64UrlDecode(string input, int offset, char[] buffer, int bufferOffset, int count)
```

Comment:
`buffer` isn't needed, because there is no intermediate step `input -> base64` (which would be in `buffer`).
This is OK according to the xml comment ("Content is not preserved.").

So the checks with / for `buffer` could be eliminated?!